### ISSUE TITLE
How to Train only One Class? 

### ISSUE THREAD

**Issue by takielias** [andrewdalpino/Rubix-engine#267]
@andrewdalpino Thanks for your incredible creation. As A PHP lover, I have been expecting this type of project for a long time. 

Though I'm new to ML, I need to train in only one label. I got the [CIFAR-10](https://github.com/RubixML/CIFAR-10) or the [MNIST](https://github.com/RubixML/MNIST) example. But I could not manage it. It's required multiple labels. How can I do it? Like I need to detect only pigeons in the image.

**Comment by raijyan** [comment]
In your example you still have two classes. "Contains pigeon", "Doesn't contain pigeon". You'd label your dataset as such.

**Comment by takielias** [comment]
> In your example you still have two classes. "Contains pigeon", "Doesn't contain pigeon". You'd label your dataset as such.
@raijyan 
Yes, I made 2 folders, 1. Pigeon, 2. None. But How should I define none? How to put everything else pigeon in none folder?

**Comment by andrewdalpino** [comment]
Hey thanks @takielias I'm just reading this now! It sounds like you want image segmentation? In other words, you want the model to draw a bounding box around the pigeons in an image? If so, then what you need is a multimodal (classifier + regressor) model that A. predicts the class of what's in the image and B. draws a bounding box around the object (usually this is done by predicting the top left and bottom right coordinates).

Unfortunately, Rubix ML does not offer multimodal prediction at the moment.

Thanks for your kind words, I appreciate your support.

**Comment by takielias** [comment]
Yes, exactly. Waiting for the multi-model support of RubixML.
